cmd/conf/chain.go

```diff
@@ -38,17 +38,21 @@ func (c *L1Config) ResolveDirectoryNames(chain string) {
 }
 
 type L2Config struct {
-	ChainID   uint64                   `koanf:"chain-id"`
-	DevWallet genericconf.WalletConfig `koanf:"dev-wallet"`
+	ChainID        uint64                   `koanf:"chain-id"`
+	ChainInfoFiles []string                 `koanf:"chain-info-files"`
+	DevWallet      genericconf.WalletConfig `koanf:"dev-wallet"`
 }
 
 var L2ConfigDefault = L2Config{
-	ChainID:   0,
-	DevWallet: genericconf.WalletConfigDefault,
+	ChainID:        0,
+	ChainInfoFiles: []string{},
```

Comment:
Ideally I think we'd pull in `arbitrum_chain_info.json` somehow, but finding the path at runtime is tricky and Go doesn't have nice support for pulling in files at compile time. We should discuss as a team the best approach here.

Reply:
done.